Assignment 2/180501097/attendanceStudents.py

```diff
@@ -0,0 +1,8 @@
+n=int(input('Enter the No of Students : '))
+attendance=[]
+print('Enter your string : ')
+for i in range(n):
+    b=input().split('-')
+    if b[1] == 'P':
+        attendance.append(b[0])
+print('People who were present are : '+str(attendance))
```

Comment:
str(list) will not give you an output like 
```
string1 string2 string3 
```
 it will give you an output like 
```
['string1', 'string2', 'string3'] 
```
change that to get the specified output